dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Plc::EnrollmentEvaluationsController < ApplicationController
+  def perform_evaluation
+    authorize! :read, Plc::Course
+    @plc_user_course_enrollment = Plc::UserCourseEnrollment.find(params[:enrollment_id])
+    @questions = @plc_user_course_enrollment.plc_course.plc_evaluation_questions.all
+    @user = @plc_user_course_enrollment.user
+    @course = @plc_user_course_enrollment.plc_course
```

Comment:
Nit: it doesn't look like `@plc_user_course_enrollment` or `@user` need to be instance variables.  (Is `@user` used anywhere?)